src/webxdc.rs

```diff
@@ -2568,4 +2568,42 @@ sth_for_the = "future""#
         );
         Ok(())
     }
+
+    /// Tests extensibility of WebXDC updates.
+    ///
+    /// If an update sent by WebXDC contains unknown properties,
+    /// such as `a_new_unknown_property` or a reserved property
```

Comment:
```suggestion
    /// such as `aNewUnknownProperty` or a reserved property
```

js uses camel case, let's not mix that as things may be used as a pattern (`max_serial` is sneaked in the same way, rust-ppl doing js :)